.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -43,17 +43,47 @@ If you need to submit a pr for any of these repos during this period, you should
 
 - [ ] **Enable code freeze bot** In order to enable the code freeze GitHub you need to change the line `if: github.repository_owner == 'adoptium' && false` to be `if: github.repository_owner == 'adoptium' && true` in the [code-freeze.yml](https://github.com/adoptium/.github/blob/main/.github/workflows/code-freeze.yml#L21) GitHub workflow. Please contact the PMC if you need help merging this change.
 - [ ] **Disable nightly testing** to free up resources and ensure no competing jobs during release week
```

Comment:
We spoke recently about disabling nightly builds completely while the initial pipelines are in progress (so likely just for 2-3 days) - do we want to implement this instead? And if so it should possibly be in the release week checklist instead.

Reply:
yes added